vim25/soap/client.go

```diff
@@ -159,6 +159,13 @@ func NewClient(u *url.URL, insecure bool) *Client {
 	return &c
 }
 
+type CertVerifyFunc func(rawCerts [][]byte, verifiedChains [][]*x509.Certificate) error
+
+func (c *Client) SetCertVerificationMethod(verifyMethod CertVerifyFunc) {
```

Comment:
Thanks @annamooseity .  It is possible to set this field currently using:
```go
soap.Client.Transport.(*http.Transport).TLSClientConfig
```

If we were to add something else that helps simplify, how about a more general method such as:
```go
func (c *Client) DefaultTransport() *http.Transport {
   return c.t
}
```
This would give users simpler access to all Transport fields. What do you think?
There are a few other places in the repo that could use the helper as well:
```
govc/about/cert.go:102:	t := c.Client.Transport.(*http.Transport)
govc/flags/client.go:311:	if t, ok := sc.Transport.(*http.Transport); ok {
govc/flags/host_connect.go:78:			t := c.Transport.(*http.Transport)
session/manager.go:127:		c.Transport.(*http.Transport).TLSClientConfig.InsecureSkipVerify = true
vapi/library/library.go:127:				t := c.Transport.(*http.Transport)
vapi/library/library_item_updatesession_file.go:119:		if c.Transport.(*http.Transport).TLSClientConfig.InsecureSkipVerify {
```

Reply:
I really like this! When I have time, I'd like to put up a new PR (or re-open this one, whichever you prefer) with your suggested changes.

Reply:
Thanks @annamooseity , whichever is easier for you is fine with me!